templates/layout.js

```diff
@@ -0,0 +1,15 @@
+
+module.exports = (title, body, scripts) => `
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <link rel="stylesheet" href="/style.css">
+      <title>${title}</title>
+    </head>
+    <body>
+    ${body}
+    </body>
+    ${scripts}
```

Comment:
probably inconsequential but do script tags usually go inside the body tag or outside?